src/dnaio/_core.pyx

```diff
@@ -215,28 +215,80 @@ cdef class Sequence:
 
 
 cdef class BytesSequence:
+    """
+    A sequencing read with read name/id and (optional) qualities
+
+    If qualities are available, they are as
+    For a Sequence a FASTA file
+    record containing a read in a FASTA or FASTQ file. For FASTA, the qualities attribute
+    is None. For FASTQ, qualities is a bytes object and it contains the qualities
+    encoded as ASCII(qual+33).
```

Comment:
That text is messed up. Just leave it out and I’ll fix it up in both Sequence and BytesSequence when I go through the docstrings.

Reply:
Whoops. Shouldn't have copied that ad verbatim indeed. Removed that paragraph now.